src/Tribe/Event_Repository.php

```diff
@@ -100,13 +149,9 @@ public function filter_by_cost( $value, $operator = '=', $symbol = null ) {
 						{$prefix}_ticket_cost.meta_key REGEXP %s
 						OR {$prefix}_ticket_cost.meta_id IS NULL
 					)
-			)",
-			$price_regexp
-		) );
+			)", $price_regexp ) );
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: d6c44840a8a6221f43b3e945b87b9859</sub>